samplesheets/assayapps/generic/plugins.py

```diff
@@ -0,0 +1,175 @@
+"""Assay app plugin for samplesheets"""
+
+from django.conf import settings
+
+from samplesheets.plugins import SampleSheetAssayPluginPoint
+from samplesheets.rendering import SIMPLE_LINK_TEMPLATE
+from samplesheets.utils import get_top_header
+from samplesheets.views import MISC_FILES_COLL, RESULTS_COLL
+
+
+# Local constants
+APP_NAME = 'samplesheets.assayapps.generic'
+RESULTS_REPORTS_COMMENT = 'SODAR Assay Plugin ResRep'
+MISC_FILES_COMMENT = 'SODAR Assay Plugin MiscFiles'
+DATA_COMMENTS = [
```

Comment:
We probably don't want to use a fixed amount of hardcoded values here. Maybe this variable could define the prefix (e.g. `SODAR Assay Plugin Data`) and then we could iterate for as long the prefix + a running integer is found in the comments.

Reply:
implemented